tst/EnergyPlus/unit/VariableSpeedCoils.unit.cc

```diff
@@ -3008,4 +3012,852 @@ TEST_F(EnergyPlusFixture, VariableSpeedCoils_ContFanCycCoil_Test)
     EXPECT_NEAR(state->dataVariableSpeedCoils->VarSpeedCoil(DXCoilNum).OutletAirHumRat, 0.00810, 0.00001);
     EXPECT_NEAR(state->dataVariableSpeedCoils->VarSpeedCoil(DXCoilNum).OutletAirEnthalpy, 25597.58, 0.01);
 }
+
+TEST_F(EnergyPlusFixture, VariableSpeedCoils_RatedSource_Temp_ASHP_Cooling)
+{
```

Comment:
New unit tests to verify that the new function that determined the rated source temperature provides the correct value for all cases currently taken into account under the variable speed coils code.